contracts/contracts/interfaces/IProviderRegistry.sol

```diff
@@ -10,7 +10,7 @@ interface IProviderRegistry {
     event FundsDeposited(address indexed provider, uint256 amount);
 
     /// @dev Event emitted when funds are slashed
-    event FundsSlashed(address indexed provider, uint256 amount);
+    event FundsSlashed(address indexed provider, uint256 amount, bytes32 indexed commitmentDigest);
```

Comment:
If we add a new field which is indexed I am not sure if it will be able to Watch/Filter this event if we listen for it for block numbers before the contract update was made. This might be a problem for indexing the chain data using other tools.